flake.nix

```diff
@@ -97,13 +109,73 @@
           hooks = {
             nixpkgs-fmt.enable = true;
             deadnix.enable = true;
+            purs-tidy.enable = true;
           };
+          tools = { inherit (ormoluLive) purs-tidy; };
         };
+
+        ormoluLive =
```

Comment:
What do you think about putting this in `default.nix` in `ormolu-live` similar to how it is done for some tests? Isolating Ormolu Live-related Nix expressions could perhaps improve readability of both `flake.nix` and `ormolu-live/default.nix`.